test/source/tests/browser-unit-tests/unit-Mime.js

```diff
@@ -320,21 +320,10 @@ BROWSER_UNIT_TEST_NAME(`Mime attachment file names`);
 
 BROWSER_UNIT_TEST_NAME(`Mime attachment file name issue 3352`);
 (async () => {
-  const originalName = 'XX J 1 IT E (P 4) p_c.pdf';
-  const attachments = [new Attachment({ name: originalName, type: 'text/plain', data: new Uint8Array([80, 81]) })];
-  const encoded = await Mime.encode({ 'text/plain': 'text' }, { Subject: 'subject' }, attachments);
-  const decoded = await Mime.decode(encoded);
-  if (decoded.attachments.length !== 1) {
-    throw Error(`Decoded MIME message has unexpectedly ${decoded.attachments.length} attachments, expecting 1`);
-  }
-  const index = 0;
-  const extractedAttachment = decoded.attachments[index];
-  if (typeof extractedAttachment === 'undefined') {
-    throw Error(`could not extract attachment at index ${index}`);
-  }
-  const extractedName = extractedAttachment.name;
-  if (extractedName !== originalName) {
-    throw Error(`extractedName unexpectedly ${extractedName}, expecting ${originalName}`);
-  }
-  return 'pass';
+  return await Mime.checkAttachmentDecode('XX J 1 IT E (P 4) p_c.pdf');
+})();
+
+BROWSER_UNIT_TEST_NAME(`Mime attachment file name issue #3505`);
+(async () => {
+  return await Mime.checkAttachmentDecode('what\'s_up?.txt');
```

Comment:
not quite clear how this test covers `XssSafeFactory.srcPgpAttachmentIframe` method.
Can you please provide a `eml`-formatted message with this attachment? We can then convert it to our `json` format for testing.

Reply:
if you have it in a gmail inbox, it's even easier. EML is not needed. You need to have Flowcrypt extension installed for this account and...
https://github.com/FlowCrypt/flowcrypt-browser/blob/fae2bb7d00555a53713cb13c2300b54ff29c3aa6/test/source/mock/google/google-data.ts#L102:L106

Reply:
```
   * This is the proper way to add messages to mock api for testing:
   *   1) log into flowcrypt.compatibility@gmail.com
   *   2) go to Settings -> Inbox and find your message
   *   3) click "download api export"
   *   4) save the json file to exported-messages folder
```

Reply:
Got it. Thank you